lib/telegram/bot/client.rb

```diff
@@ -34,8 +34,10 @@ def fetch_updates
           update = Types::Update.new(data)
           @options[:offset] = update.update_id.next
           message = update.current_message
-          log_incoming_message(message)
-          yield message
+          unless message.text.nil?
+            log_incoming_message(message)
+            yield message
```

Comment:
As far as I understand some kind of messages might have `nil` text. If I don't miss anything it is the case for voice messages and maybe something else.
With this code we wouldn't `yield` them to the provided block, so, those messages couldn't handled at all, could them?